app/assets/javascripts/charts.js

```diff
@@ -65,9 +67,15 @@ function expanded_pie(target, data){
         show: true,
         innerRadius: 0.8*3/4,
         radius: 0.8,
-        labels: {
```

Comment:
IMO, keep them both may result in lack of consistency, plus make the tooltip irrelevant. 
But for a smarter comparison, I'll try to do so.